drydock/plugin.py

```diff
@@ -2,7 +2,7 @@
 import functools
 import os
```

Comment:
Please add the Python [support](https://github.com/eduNEXT/drydock/blob/main/setup.py#L53) for 3.11 and 3.12 as [Tutor](https://github.com/overhangio/tutor/blob/master/setup.py#L75) does